website/components/_Controls.tsx

```
@@ -59,8 +59,32 @@ const snippet = (live: Live) => {
+      return;
+    }
     const { data } = event;
+    if (
+      typeof data !== "object" || data === null ||
+      typeof (data as { type?: unknown }).type !== "string"
+    ) {
+      return;
+    }
     switch (data.type) {
       case "editor::inject": {
         return eval(data.args.script);
```

Biome (2.4.14)

[error] eval() exposes to security risks and performance issues.

(lint/security/noGlobalEval)